### PR TITLE
[docs-only] update docs structure

### PR DESCRIPTION
**Pull request by felixboehm**
- update pathes to monorepo
- renamed phoenix to web (docs only) - reverted mainly, to keep flaex CI compatible
- init development and deployment section
- several small doc improvements

### CODE REVIEW

File: docs/ocis/development/building-docs.md [hunk 1]
```diff
@@ -26,9 +26,15 @@ hugo -D server
 
 Then open "http://localhost:1313/"
 
-When making changes to the docs, run `make docs` again and the server will pick up the changes and reload the page automatically
+When making changes to the docs, run 
+
+```bash
+rsync -ax --delete ../docs/ content/
+```
```
Comment (micbar) [inline docs/ocis/development/building-docs.md:33]: needs to be `rsync -ax --delete ../docs/ content/ocis`

**Review by micbar — COMMENTED** [review]
See comment. Rest looks fine

**Review by micbar — CHANGES_REQUESTED** [review]
@felixboehm Actually the rename from `phoenix` to `web` in the extensions folder is too early. The CI needs the same name like the Extension name.

File: docs/ocis/development/building-docs.md (hunk 1 above)
Comment (felixboehm) [reply docs/ocis/development/building-docs.md:33]: no. run this in folder ocis/hugo to get all docs moved to hugo/contents.
docs/ocis, docs/extensions and docs/_index.md

Comment (micbar) [reply docs/ocis/development/building-docs.md:33]: sorry, was confused. Was not aware that you are talking about the root level

**Comment by micbar** [comment]
Seems something went wrong during the rebasing.

**Pull request closed without merging**